terraform/projects/app-ecs-instances/main.tf

```diff
@@ -56,7 +56,7 @@ variable "ecs_instance_root_size" {
 variable "ecs_instance_ssh_keyname" {
   type        = "string"
   description = "SSH keyname for ECS container instances"
-  default     = "ecs-monitoring-ssh-test"
+  default     = "djeche-insecure"
```

Comment:
probably shouldn't have your name in here

Reply:
changed, thanks for spotting that :D